server/service/src/requisition/request_requisition/add_from_master_list.rs

```diff
@@ -127,6 +129,7 @@ fn generate(
         store_id,
         &requisition_row,
         items_ids_not_in_requisition,
+        HashMap::new(),
```

Comment:
this is the change for item to new line id mapping, see generate_requisition_lines